### PR TITLE
Add automated accessibility tests

### PR DESCRIPTION
**Pull request by mforner13**
## Purpose of PR

This PR adds the [cypress-axe package](https://github.com/component-driven/cypress-axe) with its dependency axe-core to the repository to allow for automated accessibility testing, and adds a test for accessibility issues to the home page spec. 

I am adding this to the repository now, so that accessibility issues can be addressed whilst building, rather than retrospectively. 

### How to use cypress-axe

1. Add the command `cy.injectA11y()` after wherever `cy.visit(url)` is called. This injects the `axe-core` runtime into the page being tested.
2. Add a test to check for accessibility which includes the command `cy.runA11y()`. This is a custom command which will output details of any accessibility violations into a table in the terminal where cypress is running, as in the screenshot below. I added a separate table which links to the documentation for each accessibility error as including another column in the first table made the final column run onto two rows and was very hard to read.

Note: Details of the `context` and `options` arguments which can be passed to `cy.runA11y()` can be found in the [cypress-axe readme](https://github.com/component-driven/cypress-axe#cychecka11y).

Screenshot 2021-04-13 at 14 40 44

### The PR also includes

- Updates to some packages which had more recent versions
- Installation of cypress version 6.9.1, as version 7 (previously installed) is not yet compatible with cypress-axe
- Fixes for accessibility issues on the index and base templates which the accessibility tests picked up
- An update to the readme on how to include automated accessibility testing in a cypress test

### CODE REVIEW

**Review by NickFitz — APPROVED** [review]
Seems good to me. I pulled the branch down locally, added a couple of obvious accessibility issues to the home page, and was suitably chastised when running the tests. 👍

File: README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ The project uses a `Makefile` to make running commands easier. `make` commands n
 - The `AUTHBROKER_CLIENT_ID` and `AUTHBROKER_CLIENT_SECRET` values needed for staff SSO can be found in [passman](https://passman.ci.uktrade.digital/2fa/verify/?next=/secret/61f0a3bf-33f3-427e-8ade-cdee0c637031/)
 
 ### Setting up static files
-The project uses (webpack)[https://webpack.js.org/] to build static files, to setup:
+The project uses [webpack](https://webpack.js.org/) to build static files, to setup:
```
Comment (dev-digital) [inline README.md:19]: I have the same commit on my branch.. its going to frown when I merge :sweat:

**Review by dev-digital — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (mforner13) [reply README.md:19]: Ahh great minds think alike :p 

I'd have thought it won't show any conflicts if it's the same change though? :)

**Pull request merged**